contracts/facets/DFMoveFacet.sol

```diff
@@ -44,7 +44,7 @@ contract DFMoveFacet is WithStorage, WithArenaStorage {
         uint256 abandoning
     );
 
-    event GameStarted(address player);
+    event GameStarted(uint256 timestamp);
```

Comment:
for what it's worth, we will need to change this for multiplayer matches because game shouldn't start when one player decides, but ok for now.